### ISSUE TITLE
Buid leosatellites error

### ISSUE THREAD

**Issue by ChengFuWeiWei** [Avian688/leosatellites#15]
Enviornment:
Ubuntu VM `16.04 LTS`
Omnetpp `5.6.2`
inet `4.2.5`

When I ran the following command
1.`cd mysamples/leosatellites/src`
2. `opp_makemake --make-so -f --deep -O out -KINET_PROJ=$HOME/omnetpp-5.6.2/mysamples/inet4 -KOS3_PROJ=$HOME/omnetpp-5.6.2/mysamples/os3 -DINET_IMPORT '-I$(OS3_PROJ)/src' '-I$(INET_PROJ)/src' -I/usr/include/curl '-L$(INET_PROJ)/src' '-L$(OS3_PROJ)/src' '-lINET$(D)' '-los3$(D)'`
3. `make MODE=release && make MODE=debug`

I got this error:
`networklayer/configurator/ipv4/LeoIpv4NetworkConfigurator.cc:196:53: error: cannot convert ‘igraph_vector_int_t*’ to ‘const igraph_integer_t* {aka const int*}’ for argument ‘2’ to ‘int igraph_vector_int_init_copy(igraph_vector_int_t*, const igraph_integer_t*, long int)’`

my igraph version is `0.9.5`
Is it the igraph version error or other information I might need ?

**Comment by Avian688** [comment]
Yes, I am using the igraph develop branch as there is some functionality that was needed (k-shortest-paths). Let me know if this fixes the issue!

**Comment by Avian688** [comment]
I am in the process of updating to OMNeT++ 6.0, so expect some changes soon.

**Comment by ChengFuWeiWei** [comment]
I changed my igraph version to `develop` that can fixes this problem.

**Comment by ChengFuWeiWei** [comment]
When I run with `release` mode 
got the following error:
`<!> Error: Cannot load library '/home/wei/omnetpp-5.6.2/mysamples/leosatellites/src//libleosatellites.so': /home/wei/omnetpp-5.6.2/mysamples/leosatellites/src//libleosatellites.so: undefined symbol: igraph_vector_init`

**Comment by Avian688** [comment]
I have updated all of the code (and its dependencies) to OMNeT++ 6.0 and INET 4.4. I have also uploaded an igraph project which you should be able to use without issues. Ensure that the igraph dependencies are included - which is depicted on the README file of the leosatellites project. Hope this helps.

**Comment by LQZhong** [comment]
> 

Hey Weiwei, have you solved this error? I have the same error with libigraph dev 0.9.6 in Ubuntu 22.04
